src/cinder/Log.cpp

```diff
@@ -502,6 +521,64 @@ void LoggerSysLog::write( const Metadata &meta, const string &text )
 	syslog( sysLevel , "%s %s", meta.toString().c_str(), text.c_str() );
 }
 
+#elif defined( CINDER_MSW )
+
+// ----------------------------------------------------------------------------------------------------
+// MARK: - LoggerEventLog
+// ----------------------------------------------------------------------------------------------------
+
+LoggerEventLog::LoggerEventLog()
+{
+	char filename[MAX_PATH];
+	wchar_t wFilename[MAX_PATH];
+	string stem;
+
+	DWORD size = GetModuleFileNameA( NULL, filename, MAX_PATH );
+	if( size ) { 
+		boost::filesystem::path exePath(filename);
```

Comment:
Here we probably want to use `fs::path`, since that'll eventually turn into the C++17 std::filesystem::path. That is already used for Visual Studio 2015 (or vc140), see [here](https://github.com/cinder/Cinder/blob/glNext/include/cinder/Filesystem.h#L38-L44) for details.